packages/xo-server/src/schemas/log/jobStart.js

```diff
@@ -16,6 +16,12 @@ export default {
     key: {
       type: 'string',
     },
+    mode: {
+      enum: ['full', 'delta'],
+    },
```

Comment:
This is specific to `backup` jobs, I don't think we should define it here, maybe in a nested value.

Reply:
In `data: {}` ?

Reply:
No,  i mean i add the data property to the `jobStart` log.

Reply:
Why not :)